pgoapi/pgoapi.py

```diff
@@ -541,11 +541,22 @@ def cleanup_pokemon(self, inventory_items=None):
         caught_pokemon = self.get_caught_pokemons(inventory_items)
         for pokemons in caught_pokemon.values():
             if len(pokemons) > self.MIN_SIMILAR_POKEMON:
-                # highest lvl pokemon first
-                sorted_pokemons = sorted(pokemons, key=self.pokemon_lvl, reverse=True)
-                for pokemon in sorted_pokemons[self.MIN_SIMILAR_POKEMON:]:
-                    if self.is_pokemon_eligible_for_transfer(pokemon, sorted_pokemons[0]):
-                        self.do_release_pokemon(pokemon)
+                if self.RELEASE_DUPLICATES:
```

Comment:
can we move logic for deciding what pokemon to keep or not  to the is_pokemon_eligible to transfer. Want to keep things clean.